processing/src/main/java/io/druid/query/aggregation/DoubleMaxAggregatorFactory.java

```diff
@@ -55,25 +57,40 @@ public DoubleMaxAggregatorFactory(String name, String fieldName)
   @Override
   public Aggregator factorize(ColumnSelectorFactory metricFactory)
   {
-    return new DoubleMaxAggregator(getDoubleColumnSelector(metricFactory, Double.NEGATIVE_INFINITY));
+    DoubleColumnSelector doubleColumnSelector = getDoubleColumnSelector(metricFactory, Double.NEGATIVE_INFINITY);
+    return NullHandlingHelper.getNullableAggregator(
```

Comment:
Could AggregatorFactory's API be changed so that this boilerplate shouldn't be copied in each AggregatorFactory (what about third-party ones?), but outside of them, in AggregatorFactory's use sites?

Reply:
Moving null handling outside of AggregatorFactories would also allow to make it non-static, that I expect will quickly become a requirement, e. g. allow to override the default null handling in a specific query.